frontend/device/remarkable/device.lua

```diff
@@ -21,24 +22,27 @@ local Remarkable = Generic:new{
 }
 
 local Remarkable1 = Remarkable:new{
-    model = "reMarkable",
     mt_width = 767, -- unscaled_size_check: ignore
     mt_height = 1023, -- unscaled_size_check: ignore
     input_wacom = "/dev/input/event0",
     input_ts = "/dev/input/event1",
     input_buttons = "/dev/input/event2",
-    powerd = "device/remarkable/powerd_rm1",
+    -- TODO: older firmware doesn't have the -0 on the end of the file path
```

Comment:
I guess we can remove the TODO and assume most RM1 users are using a recent enough FW. Do you happen to know which is the minimum fw that has these paths?